documentation/getting_started/README.md

```diff
@@ -0,0 +1,101 @@
+# Getting Started
+
+Welcome to WoTT - the first step to securing your IoT devices.
+
+## A brief overview of WoTT's services
+
+WoTT's primary goal is to simplify and manage end-to-end security for IoT developers. 
+We provide this in a number of ways:
+
+- Simplified encryption of device communication
+- Cryptographic identity (verification of access)
+- Enabling the removal of hard coded credentials from web applications and firmware
+
+You can access all these features by installing the [WoTT Agent](https://github.com/WoTTsecurity/agent) and using the [WoTT IO dash](https://dash.wott.io) to manage your devices through a simple and user-friendly interface. We'll walk you through step-by-step in setting up and installing the WoTT agent so you can start to implement hassle-free security on your IoT devices. 
```

Comment:
Let's refer to the dashboard as "WoTT's Dashboard"

Reply:
Fixed
I missed this one oops, thanks for pointing it out.